src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -46,8 +47,9 @@ public async Task<DashboardViewModel> GetDashboardViewModelAsync(string searchTe
             if(string.IsNullOrEmpty(searchTerm)) return vm;
 
             vm.LastSearchTerm = searchTerm;
-            var searchResults = await _vacancyClient.GetSearchResultsAsync(searchTerm);
-            var searchResultsVmTasks = searchResults.Select(v => MapToViewModelAsync(v, vacancyUser)).ToList();
+            var matchedVacancyReview = await _vacancyClient.GetSearchResultAsync(searchTerm);
+            var searchResults = matchedVacancyReview != null ? new List<VacancyReview>(){ matchedVacancyReview } : new List<VacancyReview>();
```

Comment:
If we're changing the GetSearchResultAsync to return a single result we may as well chase it through to the ViewModel and the View.  So change `vm.SearchResults` to `vm.SearchResult`

Reply:
Will look into this. I was trying to avoid changing too much. The client method sounded wrong to what it was doing.